cblox/include/cblox/core/tsdf_submap.h

```diff
@@ -41,15 +41,23 @@ class TsdfSubmap {
 
   // Submap pose interaction
   const Transformation& getPose() const {
-    std::unique_lock<std::mutex> lock(transformation_mutex);
+    std::unique_lock<std::mutex> lock(transformation_mutex_);
     return T_M_S_;
   }
 
   void setPose(const Transformation& T_M_S) {
-    std::unique_lock<std::mutex> lock(transformation_mutex);
+    std::unique_lock<std::mutex> lock(transformation_mutex_);
     T_M_S_ = T_M_S;
   }
 
+  // Set interval in which submap was actively mapping
+  void startMappingTime(int32_t time) { mapping_interval_.first = time; }
```

Comment:
Would you usually use this as seconds since epoch? In some cases I'd expect something more accurate to be quite useful. How about using an `int64_t`? Compared to the size of the rest of the submap, it wouldn't be noticeable in terms of storage. And being able to store full system clock timestamps (nanosecond precision) could be quite useful :)